Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean

```diff
@@ -0,0 +1,63 @@
+/-
+Copyright (c) 2024 Antoine Chambert-Loir. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir
+-/
+
+import Mathlib.RingTheory.MvPowerSeries.Basic
+import Mathlib.Data.Finsupp.WellFounded
+
+/-! # ZeroDivisors in a MvPowerSeries ring
+
+- `mem_nonZeroDivisors_of_constantCoeff` proves that
+a multivariate power series whose constant coefficient is not a zero divisor
+is itself not a zero divisor
+
+## TODO
```

Comment:
Can you add in the documentation the counterexample showing that it is difficult to characterize zerodivisors (I mean that the analogue of [Polynomial.nmem_nonZeroDivisors_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Polynomial/RingDivision.html#Polynomial.nmem_nonZeroDivisors_iff) does not hold?

Reply:
Done.